superflore/generators/bitbake/ros_meta.py

```diff
@@ -29,30 +29,39 @@ def __init__(
             info('Creating new branch {0}...'.format(self.branch_name))
             self.repo.create_branch(self.branch_name)
 
-    def clean_ros_recipe_dirs(self, distro=None):
-        if distro:
-            info(
-                'Cleaning up meta-ros{}-{}/generated-recipes directory...'
-                .format(yoctoRecipe._get_ros_version(distro), distro))
-            self.repo.git.rm('-rf', 'meta-ros{}-{}/generated-recipes'.format(
-                yoctoRecipe._get_ros_version(distro), distro))
-        else:
-            info('Cleaning up generated-recipes directories...')
-            self.repo.git.rm('-rf', 'generated-recipes')
+    def clean_ros_recipe_dirs(self, distro):
+        info(
```

Comment:
So that the list of generated files/directories to be removed is in a single place, generate a list of them and then use it in the `info()` and `git rm -rf` calls. (A chance to use `map()`!)